gfx/Mushroom-Dream/pngs_tall_32x48/player_male.txt

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "player_male",
```

Comment:
```suggestion
    "id": [ "player_male", "player_female" ]
```

Reply:
i incorporated this, on the other PR, i'm gonna close this one because i'm afraid it'll cause trouble again 🎃